frontend/package.json

```diff
@@ -97,7 +97,7 @@
     "@ngneat/content-loader": "^7.0.0",
     "@ngx-formly/core": "^6.1.4",
     "@openproject/octicons-angular": "^19.7.0",
-    "@openproject/primer-view-components": "^0.13.1",
+    "@openproject/primer-view-components": "^0.16.0",
```

Comment:
Please also change the version in the [`overrides` section](https://github.com/opf/openproject/blob/dev/frontend/package.json#L178) at the very bottom.